SparkPost/matchbox
# UX-271 Compact Banner

### What Changed

- Created compact banner variation
- Added new prop `size` which allows `small` and `large`

### How To Test or Verify

- `npm run start:storybook`
- Visit banner story and verify small banner matches https://sparkpost.invisionapp.com/console/UI-Kit-Inventory-&-Manual-ck9ceoeh86tyk01ysw64nd9kk/ck9cepx556pev01yf03zsuoa1/play

### PR Checklist
